crates/core/src/dht/aspect_map.rs

```diff
@@ -81,7 +76,7 @@ impl AspectMap {
             .join("\n")
     }
 
-    pub fn merge(map1: &AspectMap, map2: &AspectMap) -> AspectMap {
+    pub fn merge(map1: AspectMap, map2: AspectMap) -> AspectMap {
```

Comment:
```suggestion
    pub fn merge(map1: &AspectMap, map2: &AspectMap) -> AspectMap {
```

Is there a reason to not leave these as references? It turns out that in our current codebase we always have freshly created maps to pass to this function, but that alone isn't enough reason to make this require owned values.

Reply:
In conversation I found that the `im` methods require owned values